plugins/module_utils/net_tools/pritunl/api.py

```diff
@@ -0,0 +1,307 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, Florian Dambrine <android.florian@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""
+Pritunl API that offers CRUD operations on Pritunl Organizations and Users
+"""
+
+from __future__ import absolute_import, division, print_function
+
+import base64
+import hashlib
+import hmac
+import json
+import time
+import uuid
+
+from ansible.module_utils.six import iteritems
+from ansible.module_utils.urls import open_url
+
+__metaclass__ = type
+
+
+class PritunlException(Exception):
+    pass
+
+
+def pritunl_argument_spec():
+    return dict(
+        pritunl_url=dict(required=True, type="str"),
+        pritunl_api_token=dict(required=True, type="str", no_log=False),
+        pritunl_api_secret=dict(required=True, type="str", no_log=True),
+        validate_certs=dict(required=False, type="bool", default=True),
+    )
+
+
```

Comment:
hi @Lowess this is not a problem per se, more a suggestion: function above uses `dict()` and function below uses `{}`. For the sake of consistency it might be interesting to have them both use the same construct - whichever one you prefer.

Reply:
@russoz it's just that this util function is used in modules where the `dict()` format is pretty common for `argument_spec` hence why I used `dict()` instead of `{}`

```
argument_spec = pritunl_argument_spec()

argument_spec.update(
        dict(
            organization=dict(required=True, type="str", aliases=["org"]),
            state=dict(
                required=False, choices=["present", "absent"], default="present"
            ),
            user_name=dict(required=True, type="str"),
            user_type=dict(
                required=False, choices=["client", "server"], default="client"
            ),
            user_email=dict(required=False, type="str", default=None),
            user_groups=dict(required=False, type="list", elements="str", default=None),
            user_disabled=dict(required=False, type="bool", default=None),
            user_gravatar=dict(required=False, type="bool", default=None),
        )
    ),
```

With that in mind do you still want me to change to `{}`

Reply:
hi @Lowess this is more a matter of personal preferences. I'd prefer them equal, but if you are happy with them as is, no worries, we can live with that. Thanks for the response :-)